# message to store a min-,max- and meanvalue.

float64 min
float64 max
float64 mean
